test/conformance/kuttl/filter/exclude/user/no-wildcard/policy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: block-pod
+spec:
+  validationFailureAction: Enforce
+  background: false
+  rules:
+  - name: block-pod
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    exclude:
+      any:
+      - subjects:
+        - kind: User
+          name: kubernetes-admin
```

Comment:
You know this is only going to work if tested in KinD, right?

Reply:
Yes, shall i add a note in the readme ?

Reply:
I suppose it's fine either way, I just like the idea of making these tests hermetically sealed with no tool-specific dependencies. If we wanted to switch and use, for example, K3d later on there would be no test rework needed.

Reply:
Then we should bootstrap the kubeconfig ourself with a known user we own (not dependent on the tooling).

Reply:
Yes, and that's exactly why we've done that in the kuttl tests we're making in the policy library. See example [here](https://github.com/kyverno/policies/tree/main/other/b-d/block-cluster-admin-from-ns).